### DISCUSSION TITLE
GitHub Password in Repository

### DISCUSSION THREAD

**Discussion by DT723** [community/community#23866] [New to GitHub]
<p>Can anyone tell my why my GitHub account password is being displayed in my repositorys and how to remove it? It’s sitting in the header of the MAIN branch (above the files).  Not sure how it got there.  Thanks</p>

**Comment by airtower-luna** [comment]
<p>I looked at a few of your public repositories and not seeing anything resembling a password there. Could you post a screenshot? You can censor the actual password, but <em>where</em> you see it would be important.</p>

**Comment by DT723** [comment]
<p>In my portfolio.  Maybe only I can see it??</p>

**Comment by DT723** [comment]
<p><img src="https://user-images.githubusercontent.com/70732710/181079765-d5601b27-71c4-4f05-b01a-8a235fe7a4aa.png" alt="gitp" data-base62-sha1="5ue1mClWI1edg3Erh49iXpRibzt" width="687" height="118"></p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Ah, now I can see what’s wrong there. What’s displayed there is the <em>author name</em> set in that commit. And looking at the history of that repository, there seem to be a lot of commits with that author name. The email address in those commits looks wrong too. :wink:</p>
<p>Things you should do now (in order):</p>
<ol>
<li>Change your password. :warning:
</li>
<li>Set the correct author name and email address on the computer where you made those commits (and maybe check if they’re correct on any other computer you use). See the “Your Identity” section in <a href="https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup#_your_identity" class="inline-onebox">Git - First-Time Git Setup</a> for how to do that.</li>
<li>If you want to, you can go back and edit the affected commits to fix the author information. See <a href="https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History" class="inline-onebox">Git - Rewriting History</a>.</li>
</ol>
<p>Good luck!</p>

**Comment by DT723** [comment]
<p>oh wow, I didn’t realize what was happening.  Thanks for the help, I will definitely spend some time in GitHub to fix this. Thanks again!</p>
